### PR TITLE
Development

### PR DESCRIPTION
**Pull request by alemmaz**
-acomodando un poco header, footer y main
-rama development creada

### CODE REVIEW

**Review by nathan-hyan — APPROVED** [review]
🥇

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "liveServer.settings.port": 5501
```
Comment (nathan-hyan) [inline .vscode/settings.json]: Add this file to a `.gitignore` file. Ask me if you need help with this

File: index.html [hunk 2]
```diff
@@ -10,66 +10,92 @@
...
     <header class="title">
         <h1>Bodega de Comics</h1>
     </header>
+    <div class="slidecontainer">
+    <ul class="slider">
+        <li id="slide1"><img src="img/slider1.jpg" alt="" class="imgslider"></ li>
+        <li id="slide2"><img src="img/slider2.jpg" alt="" class="imgslider"></ li>
+        <li id="slide3"><img src="img/slider3.jpg" alt="" class="imgslider"></ li>
+        <li id="slide4"><img src="img/sli2.jpg" alt="" class="imgslider"></ li>
+        <li id="slide5"><img src="img/sli1.jpg" alt="" class="imgslider"></ li>
```
Comment (nathan-hyan) [inline index.html]: Put the description of the image in the `alt` prop. Don't leave it empty

File: index.html [hunk 3]
```diff
@@ -10,66 +10,92 @@
...
+     </ul>
+    </div> 
 
+     <nav>
+        <ul class="menu">
+            <li><a href="#slide1">1<//a></ li>
+            <li><a href="#slide2">2<//a></ li>
+            <li><a href="#slide3">3<//a></ li>
+            <li><a href="#slide4">4<//a></ li>
+            <li><a href="#slide5">5<//a></ li>
```
Comment (nathan-hyan) [inline index.html]: Fix the `<//a>` typo here, it's `</a>`

File: index.html [hunk 4]
```diff
@@ -10,66 +10,92 @@
...
-        <div class="related-post">
-            another post
-        </div>
-        <div class="related-post">
-            another post
-        </div>
-        <div class="related-post">
-            another post
-        </div>
+
```
Comment (nathan-hyan) [inline index.html]: same here with the `alt` in the images

File: index.html [hunk 5]
```diff
@@ -10,66 +10,92 @@
...
-        </div>
+         
 
+        <div class="relatedPostParent">
+            
+                <img src="img/footer1.jpg" alt="" class="comicsG"> 
+                <img  src="img/footer2.jpg" alt="" class="comicsG">
+                <img  src="img/footer3.jpg" alt="" class="comicsG">
+                <img  src="img/footer4.jpg" alt="" class="comicsG">
+                <img  src="img/footer5.jpg" alt="" class="comicsG">
```
Comment (nathan-hyan) [inline index.html]: same here

File: style.css [hunk 6]
```diff
@@ -32,36 +105,43 @@ body {
...
-.container > .sidebar{
-    grid-column: span 2;
-} 
+
+
+
+
+
+
+
```
Comment (nathan-hyan) [inline style.css]: Use prettier so there's no excess of empty spaces. Also, remove comments when pushing code for a PR. The only comments that are permitted are `TODO: Need fixing yadayadyadyad`

**Review by nathan-hyan — CHANGES_REQUESTED** [review]
Some comments. Remember to close this PR and open a new one on a new branch once this one gets approved. So there's no further confusion.

**Comment by alemmaz** [comment]
still problem with slider, I can´t fix alignment with layout

**Review by nathan-hyan — APPROVED** [review]
Fix spacing and this is done! congrats 🥇

**Review by nathan-hyan — APPROVED** [review]
🥇

**Comment by nathan-hyan** [comment]
Close this PR, and make a new one

**Pull request merged**